docs/commands/graph.query.md

```diff
@@ -902,33 +902,35 @@ This section contains information on all supported functions from the Cypher que
+| last(_expr_)                         | Returns the last element of a list <br> Returns null when _expr_ evaluates to null or an empty list                                     
+| list.dedup(_list_) * | Given a list, returns a similar list after removing duplicate elements <br> Order is preserved, duplicates are removed from the end of the list <br> Returns null when _list_ evaluates to null <br> Error when _list_ does not evaluate to a list or to null |
+| list.insert(_list_, _idx_, _val_[, _dups_ = TRUE]) * | Given a list, returns a list after inserting a given value at a given index <br> _idx_ is 0-based when non-negative, or from the end of the list when negative <br> Returns null when _list_ evaluates to null <br> Returns _list_ when _val_ evaluates to null <br> Returns _list_ when _idx_ evaluates to an integer not in [-NumItems-1 .. NumItems] <br> When _dups_ evaluats to FALSE: returns _list_ when _val_ evaluates to a value that is already an element of _list_  <br> Error when _list_ does not evaluate to a list or to null <br> Error when _idx_ does not evaluate to an integer <br> Error when _dups_, if specified, does not evaluate to a Boolean |
+| list.insertListElements(_list_, _list2_, _idx_[, _dups_ = TRUE]) * | Given a list, returns a list after inserting the elements of a second list at a given index <br> _idx_ is 0-based when non-negative, or from the end of the list when negative <br> Returns null when _list_ evaluates to null <br> Returns _list_ when _list2_ evaluates to null <br> Returns _list_ when _idx_ evaluates to an integer not in [-NumItems-1 .. NumItems] <br> When _dups_ evaluates to FALSE: If an element of _list2_ evaluates to an element of _list_ it would be skipped; If multiple elements of _list2_ evaluate to the same value - this value would be inserted at most once to _list_ <br> Error when _list_ does not evaluate to a list or to null <br> Error when _list2_ does not evaluate to a list or to null <br> Error when _idx_ does not evaluate to an integer <br> Error when _dups_, if specified, does not evaluate to a Boolean |
+| list.remove(_list_, _idx_[, _count_ = 1]) * | Given a list, returns a list after removing a given number of consecutive elements (or less, if the end of the list has been reached). starting at a given index. <br> _idx_ is 0-based when non-negative, or from the end of the list when negative <br> Returns _null_ when _list_ evaluates to null <br> Returns _list_ when _idx_ evaluates to an integer not in [-NumItems .. NumItems-1] <br> Returns _list_ when _count_ evaluates to a non-positive integer <br> Error when _list_ does not evaluate to a list or to null <br> Error when _idx_ does not evaluate to an integer <br> Error when _count_, if specified, does not evaluate to an integer |
```

Comment:
"Error when _list_ does not evaluate to a list or to null"
Consider adding: Emit an error when ...

Reply:
Fixed